docs/commands/rhoas_kafka_update.adoc

```diff
@@ -0,0 +1,61 @@
+ifdef::env-github,env-browser[:context: cmd]
+[id='ref-rhoas-kafka-update_{context}']
+= rhoas kafka update
+
+[role="_abstract"]
+Update configuration details of a Kafka instance.
+
+[discrete]
+== Synopsis
+
+Update certain configuration details of a Kafka instance.
+
+Currently it is possible to update the "owner" field. The new owner 
+will have super-user privileges and will be authorized to manage this instance.
```

Comment:
`super-user` is not clear without explanation. I will remove that from original statement.